group-project/src/main/webapp/style.css

```diff
@@ -137,6 +137,15 @@ h2 {
   background-color: #dee1e6;
 }
 
+#commentContainer{
+    font-family: Roboto;
+    background-color: #fff;
+    font-weight: 700;
+    color: #3e4958;
+    margin: 1em 30em;
+    padding: 3em 2em;
+}
+
 input {
   width: 300px;
   height: 30px;
```

Comment:
instead of hard-coding the input element size, maybe use unit 'em'?
This makes the input shrinks with the parent element when the screen size changes.
![image](https://user-images.githubusercontent.com/91285203/178843748-c5efdfcf-ba99-4396-9307-1d9f917adf82.png)

Reply:
I thought you wrote the css for input? For the Map or on the nearbySearch-1 branch (https://github.com/matty377/summer22-sps-14/commit/eb423586d7916122d203a60d947b06ae6f8c47b4)? I can work on the css though☺️ My changes are in green; I think I copied the prototype's style.

Reply:
oh yea. That's right. I messed up there

Reply:
And yea I think it'll be great if you can to work on the CSS 😄